frontend/interfaces/platform.ts

```diff
@@ -66,3 +66,7 @@ export const HOST_LINUX_PLATFORMS = [
   "opensuse-leap",
   "opensuse-tumbleweed",
 ] as const;
+
+export const isLinuxLike = (platform: string) => {
+  return HOST_LINUX_PLATFORMS.includes(platform as any);
```

Comment:
What does this do?

Reply:
it ensures that the platform is a linux platform; the platform we get back can be many different values (e.g. ubuntu, debian, linux, arch, etc...) and this checks if it is linux like. I'll add some comments to the function.